colossalai/shardformer/README.md

```diff
@@ -390,3 +390,32 @@ To validate that training the model using shardformers does not impact its conve
 | 0.81855  | 0.87124 | 0.10357 |     1      |    False    |
 
 Overall, the results demonstrate that using shardformers during model training does not affect the convergence.
+
+### performance
+
+We also conducted [benchmark tests](./examples/performance_benchmark.py) to evaluate the performance improvement of Shardformer. We compared the training time between the original model and the shard model.
+
+In the case of using 2 GPUs, the training times are as follows:
+| N_CTX |   org_model    |  shard_model   |
+| :------: | :-----: | :-----: |
+| 256  | 10ms | 20.3ms |
+| 512  | 9.5ms | 15.9ms |
+| 1024  | 20.0ms | 19.8ms |
+| 2048  | 46.6ms | 30.0ms |
+| 4096  | 160.4ms | 89.8ms |
+
+![Alt text](./examples/images/image-2-gpus.png)
```

Comment:
Don't put this image in this folder. Create a pull request to put all images in https://github.com/hpcaitech/public_assets/tree/main/colossalai/img/shardformer. Name your images clearly e.g. `benchmark_gpus_2.png`